Updates to Super Admin Users page

briri opened the pull request:
Addresses feature requests in #1275 and #1248 
- Added `users.active` flag (defaults to true)
- Added new columns to Super Admin -> Users page
  - User Name is now a link to new `super_admin/users/[:id]/edit` page
  - Organization name
  - Created date
  - Active an ajax enabled checkbox to allow admins to quickly activate/deactivate accounts
- Added new super admin user edit page (route, controller, policy, view and JS) to allow admin to update user name, email, org and language
- Added logic to User model to enable Devise to force users who are not active to be logged out
- Updated devise's default message about account being inactive
- Updated `user_mailer` so that deactivated users do not receive emails (unless its a devise email about their account or a plan is shared with them)
- Added tests

Also:
- added logic to privilege update to only show/email users if the values have changed
- Fixed failing org controller test by setting default orgs.link value if not available 
- Added 'date-column' class to stylesheet to set min-width for date columns on users table. Usable elsewhere if necessary

**Need to run `rake db:migrate`**

vyruss commented on app/models/user.rb:
```diff
@@ -67,6 +67,12 @@ def filter(query)
+  ##
+  # This method uses Devise's built-in handling for inactive users
+  def active_for_authentication?
+    super and self.active?
```
can you please change `and` to `&&` because of this Ruby weirdness: 
https://stackoverflow.com/a/1426835/6234472

vyruss commented:
Otherwise, tested and seems fine. The only issue we may face in UAT is: 
When you edit a user and save, you have to click the back-arrow twice to go back to the list of users, and then you are not on the right paginable page

briri commented:
Updated that line of code @vyruss and also added a back button to the main users page (which doesn't retain the pagination/sort info, but is at least one less click for now)
